components/cart-page/cart-page.css

```diff
@@ -0,0 +1,76 @@
+@import url(/style.css);
+@import url(/components/color.css);
+
+.cart-page{
+    display:flex;
+    flex-wrap:wrap;
+    justify-content: center;
+    gap:4rem;
+}
+
+ .cart-page{
+     margin-top:9rem;
+     margin-bottom:2rem;
+ }
+
+  .btn-cart{
+      background-color: var(--pink-primary2);
+      font-size:1rem;
+      height:2rem;
+  }
+
+   .btn-cart:nth-of-type(2){
+     color:var(--pink-primary2);
+     background-color:var(--white);
+     border:1.5px solid var(--pink-primary2);
+   }
+
+  .price-table{
+      border:1px solid color;
+  }
+
+  .btn-cart:nth-of-type(1):hover{
+    background-color: var(--pink-primary1);
+  }
+
+  .btn-cart:nth-of-type(2):hover{
+    background-color: var(--pink-light2);
+  }
+
+  .price-table-h th, td,tr{    padding:1rem;
```

Comment:
Suggestion: use write indentation in file, it will help in readability

Reply:
i have made the change, thanlyou for review